addons/fastroping/functions/fnc_fastRopeLocalPFH.sqf

```diff
@@ -31,10 +31,19 @@ if (animationState _unit != "ACE_FastRoping") exitWith {
     [_unit, "ACE_FastRoping", 2] call EFUNC(common,doAnimation);
 };
 
+[_unit] spawn {
```

Comment:
I think PFH is fine, it's only every 1 second and less complicated than waitAndExecute.